src/address/types.rs

```diff
@@ -224,10 +224,15 @@ impl AsRef<OsStr> for Address {
 #[derive(Debug, PartialEq, Clone, Copy)]
 /// Errors in email addresses parsing
 pub enum AddressError {
+    /// Missing domain or user
     MissingParts,
+    /// Unbalanced angle bracket
     Unbalanced,
+    /// Invalid email user
     InvalidUser,
+    /// Invalid email domain
     InvalidDomain,
+    /// Invalid UTF8b data
     InvalidUtf8b,
```

Comment:
Now that I look at it I don't think we are constructing this variant anymore, since starting from #607 we never decode the utf8b data.